src/python/pants/backend/codegen/antlr/python/antlr_py_gen.py

```diff
@@ -0,0 +1,105 @@
+# coding=utf-8
+# Copyright 2016 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import logging
+import os
+
+from pants.backend.codegen.antlr.python.python_antlr_library import PythonAntlrLibrary
+from pants.backend.jvm.targets.jar_dependency import JarDependency
+from pants.backend.jvm.tasks.nailgun_task import NailgunTask
+from pants.backend.python.targets.python_library import PythonLibrary
+from pants.base.exceptions import TaskError
+from pants.option.custom_types import target_option
+from pants.task.simple_codegen_task import SimpleCodegenTask
+from pants.util.dirutil import safe_mkdir, touch
+from pants.util.memo import memoized_method
+
+
+logger = logging.getLogger(__name__)
+
+
+_ANTLR3_REV = '3.1.3'
```

Comment:
There surely is, but the goal here was just parity with the old Python ANTLR builder, so we can kill it. Having this be configurable would be best done as part of merging the implementations of this and the Java ANTLR gen task, thus supporting ANTLR4, multiple versions, etc etc.